backend/user_management/minio/routers/upload.py

```
@@ -1,41 +1,125 @@
+        file_urls = getattr(product, product_field, [])
+        for url in file_urls:
+            if file_name in url:
+                return product, url
+    return None, None
+
+@router.delete("/upload/{upload_type}/{file_name}", response_model=ProductPhotoUploadResponse)
+async def delete_uploaded_file(
+    upload_type: str,
+    file_name: str,
+    db: Session = Depends(get_db),
+    user: User = Depends(get_current_user)
```

Ruff (0.8.2)

Unused function argument: `user`

(ARG001)